substrate/socket

```diff
@@ -98,6 +98,52 @@ namespace substrate
 	{
 		SUBSTRATE_API sockaddr_storage prepare(const socketType_t family, const char *const where,
 			const uint16_t port, const socketProtocol_t protocol = socketProtocol_t::tcp) noexcept;
+
+		inline int typeToFamily(const socketType_t type) noexcept
+			{
```

Comment:
The function {}'s and body need de-indenting by 1 level to match code style (same for the other 3 below)

Reply:
I recall from earlier reviews, that your code style requires indenting body by one tab. Please confirm that this is the case.

Reply:
Single-line body functions like the one on line 79/80 should be, but normal functions are not.